contrib/review-trailers-tooling/README.md

```diff
@@ -0,0 +1,41 @@
+## What's this?
+
+A basic set of tools to ease adding review trailers, in order to acknowledge reviewers' work.
+
+## Which trailers are to be added?
+
+- `Helped-by: Full Name <email@address>`
+   Added by regular users to particular commits within a PR. Applies to the commit it appears on. Means "this commit was somehow improved based on suggestions by @reviewer".
+- `Reviewed-by: Full Name <email@address>`
+   Added by maintainer to PR merge commit. Applies to whole PR. Means "these changes were carefully reviewed by @reviewer an he has stated conformity with them."
+
+## Which tools are included, and how they help?
+
+- Command line tool to transform GitHub usernames into `Full Name <email@address>` info.
```

Comment:
Ah, I see. In that case I think `A command-line tool ...` works better.

On March 29, 2015 5:36:37 PM EDT, "Eliseo Martínez" <notifications@github.com> wrote:
>> @@ -0,0 +1,41 @@
>> +## What's this?
>> +
>> +A basic set of tools to ease adding review trailers, in order to
>acknowledge reviewers' work.
>> +
>> +## Which trailers are to be added?
>> +
>> +- `Helped-by: Full Name <email@address>`
>> +   Added by regular users to particular commits within a PR. Applies
>to the commit it appears on. Means "this commit was somehow improved
>based on suggestions by @reviewer".
>> +- `Reviewed-by: Full Name <email@address>`
>> +   Added by maintainer to PR merge commit. Applies to whole PR.
>Means "these changes were carefully reviewed by @reviewer an he has
>stated conformity with them."
>> +
>> +## Which tools are included, and how they help?
>> +
>> +- Command line tool to transform GitHub usernames into `Full Name
><email@address>` info.
>
>That refers to `github-user-info` executable, which is a single
>command-line tool.
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/neovim/neovim/pull/2291/files#r27359417